### DISCUSSION TITLE
ping pong  in version 4.4.1 not work Automatically

### DISCUSSION THREAD

**Discussion by elelemy** [socketio/socket.io#4314] [Q&A]
the server send ping to client only once and the client responds but after that the server doesn't ping again causing timeout and disconnection

the server option is 
{
	path: "/socket/events",
	transports: ['websocket'],
	allowUpgrades: false,
	pingInterval: 10000,
	pingTimeout: 25000
}

the client option is
{
	auth: {token: accessToken}, 
        transports: ['websocket'],
	path: "/socket/events",
	upgrade: false,
	autoConnect:true
}

**Comment by darrachequesne** [comment]
Hi! That sounds weird. Are you able to reliably reproduce the issue?

**Reply by elelemy** [reply]
![websocket](https://user-images.githubusercontent.com/51526451/158397667-a2738d5e-8115-42db-8fce-fd226d55e7c0.PNG)

**Reply by elelemy** [reply]
server sends ping and browser responses with pong just one time .
Then , after pingInterval + pingTimeout ,it opens a new websocket connection.

**Reply by darrachequesne** [reply]
From the screenshot above, It seems you have two different clients, one with `EIO=3` (Socket.IO v2) and one with `EIO=4` (Socket.IO v3/v4). Could that explain the issue?
